### DISCUSSION TITLE
pimcore_admin_document_document_doctypesget error on admin

### DISCUSSION THREAD

**Discussion by ghost** [pimcore/pimcore#8814] [Q&A / Support] [answered]
Hi,
after deploying new/changed stuff from dev to test-env, I got error from backend.
I see login-page for admin.
After login I get:
![pimcoreError](https://user-images.githubusercontent.com/52573912/114972049-4e8e4c00-9e9b-11eb-890d-136e70468667.png)

I deployed with:
git pull
composer install
bin/console pimcore:deployment:classes-rebuild
bin/console pimcore:cache:clear
bin/console cache:clear

Do I miss something? How do I get this error fixed?
Also Frontend is working

Any help appreciated

**Comment by MarcoGuiducci** [comment, marked as answer]
got the same problem. Any suggestion?

**Reply by ghost** [reply]
Yes, just on the Debug mode and check logs,
I had this issue because of a syntax error in my method.
**For Example:**
**Not Working** GeneralHelper::sendMail($SalesEmailAlias,'/MiSource/Emails/Customer/SalesInformNewCompany',$paramsSales,);
**Working**
GeneralHelper::sendMail($SalesEmailAlias,'/MiSource/Emails/Customer/SalesInformNewCompany',$paramsSales);

**Comment by capitalfuse** [comment]
changed to "dev" mode in .env file 
then,
execute "bin/console clear:cache" 
after that, 
execute "chown www-data:www-data ./" and confirmed to be able to login.
then 
back to "prod" mode in .env file
execute "chown www-data:www-data ./" and confirmed to be able to login. Fixed this issue.
